style.module.scss

```diff
@@ -145,6 +145,7 @@
 
 .navTabs {
   border-bottom: 1px solid #ddd;
+  margin-left: 0px !important;
```

Comment:
I just noticed that currently we have a sad margin here:

![image](https://github.com/pganalyze/pganalyze-docs/assets/911433/de6b79a1-1a73-4225-99b0-73f6e6b3856f)

This is going to help the nav tab to start without having that margin.